content/docs/06-integrations/00-spectro-k8s-dashboard.md

```diff
@@ -0,0 +1,212 @@
+---
+title: 'Spectro Kubernetes Dashboard'
+metaTitle: "Spectro Cloud's Integrated Kubernetes Dashboard"
+metaDescription: 'The Integrated Kubernetes Dashboard Monitoring pack in Palette reduces the complexity of standing up the Kubernetes dashboard for a cluster'
+hiddenFromNav: true
+type: "integration"
+category: ['integrations','monitoring']
+logoUrl: 'https://registry.spectrocloud.com/v1/k8s-dashboard/blobs/sha256:2de5d88b2573af42d4cc269dff75744c4174ce47cbbeed5445e51a2edd8b7429?type=image/png'
+---
+
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+# Spectro Kubernetes Dashboard
+
+Spectro Kubernetes Dashboard is a web-based UI for Kubernetes clusters that auto-enables the Kubernetes Dashboard. It integrates seamlessly with Spectro Proxy and uses default settings for access control and Identify Provider (IDP).   
```

Comment:
```suggestion
Spectro Kubernetes Dashboard is a web-based UI for Kubernetes clusters that auto-enables the Kubernetes Dashboard. It integrates seamlessly with [Spectro Proxy](/integrations/frp) and uses default settings for access control and Identity Provider (IDP).   
```
Typo, and added a link to the proxy to help the reader.

Reply:
Fixed typo and added link.